### ISSUE TITLE
Cannot ignore Application menu's application list in Latte Dock

### ISSUE THREAD

**Issue by talentlessguy** [esjeon/krohnkite#66]
**Symptom**
A simple description of the problem you're experiencing.

**How to Reproduce**
1. Launch latte-dock, put Application Menu there
2. Open Application Menu and then hover any app folder
3. See application list being tiled

![image](https://user-images.githubusercontent.com/35937217/77543075-2ca0e080-6eb8-11ea-8419-f0f7e213e811.png)

4. Try ignoring windows, but it doesn't help:

![image](https://user-images.githubusercontent.com/35937217/77543144-4e01cc80-6eb8-11ea-86e7-30d3d7fec98e.png)

**Expected behavior**
I expect KDE Application Menu's application list to remain on it's place, same as without Krohnkite.

tldr; I need to prevent Application menu from tiling.

**Environment**
 - Distro: Manjaro x64
 - KWin version: 5.18.3
 - Krohnkite version: 88a8f95

**Comment by esjeon** [comment]
Adding `lattedock` to "Class" should work fine. I know because I'm using Latte Dock, too. One caveat is that, once you change the script settings, you must restart the script.

Perhaps, Latte Dock should be ignored by default, because lots of people use it.

**Comment by talentlessguy** [comment]
@esjeon I have it in classes, it worked, thanks!
